packages/components/src/components/Table/Table.stories.tsx

```diff
@@ -156,7 +156,7 @@ export const WithClickableRows: Story = {
         </THead>
         <TBody>
           {map([...Array.from({ length: 100 }).keys()], (index) => (
-            <Tr isClickable key={index}>
+            <Tr isClickable isHoverable key={index}>
```

Comment:
I believe that `isHoverable` does not translate well the fact that there is an interaction on hover because technically every row is hoverable (some has interaction, some does not)

Reply:
So, what could be better? Adding a prop to the table and using it from context to change the hover of the row?

Reply:
just the name of the prop if anyone else has an idea `hasHover`?

Reply:
Thank you! Updated it here https://github.com/Localitos/pluto/pull/1385/files#diff-6b7a9252166d5be1cc435ed1af0e1469b9ef3739cbee9d45b69b15accfcdd1a5R16